tools/zetasql-helper/zetasql-toolkit-examples/src/main/java/com/google/zetasql/toolkit/examples/AddResourcesToBigQueryCatalog.java

```diff
@@ -0,0 +1,54 @@
+package com.google.zetasql.toolkit.examples;
+
+import com.google.zetasql.toolkit.catalog.bigquery.BigQueryCatalog;
+import java.util.List;
+
+public class AddResourcesToBigQueryCatalog {
+
+  public static void main(String[] args) {
+    BigQueryCatalog catalog = new BigQueryCatalog("bigquery-public-data");
+
+    // Add a table or a set of tables by name
+    // Views are considered tables as well, so they can be added this way to the catalog
+    catalog.addTable("bigquery-public-data.samples.wikipedia");
+    catalog.addTables(
+        List.of(
+            "bigquery-public-data.samples.wikipedia",
+            "bigquery-public-data.samples.github_nested"));
+
+    // Add all tables in a dataset or project
+    // Views are considered tables as well, so they will be added to the catalog too
+    catalog.addAllTablesInDataset("projectId", "datasetName");
```

Comment:
Could we have constants instead of string literals for for the projectID and datasetName. This will make it easier for a user that wants to plug in their project and dataset to do a quick test

Reply:
I don't think it makes a lot of sense for this example in particular, since it's just showcasing the methods the catalog has available for adding resources. It's not really meant to be run.

If a user was using the examples directly, they would be working with another one, such as the `AnalyzeBigQuery` example.